### DISCUSSION TITLE
SonarCloud Github Action failing

### DISCUSSION THREAD

**Discussion by dariogmori** [Arquisoft/faq#65] [Q&A]
Hi, I have found that the Github action for SonarCloud is failing for the unit-tests job.
This happens for every team as I have checked.
It might be a problem with the organization wide token.
@labra @pglez82 might want to look into it.
![Screenshot_20240416-195250_GitHub.jpg](https://github.com/Arquisoft/faq/assets/91892789/872fc74b-4cc5-4d23-99e4-ce30def51701)

**Comment by pglez82** [comment]
I do not see to get what this has to do with SonarCloud. The error that I see in the screenshot is the `chmod +x /entrypoint.sh`, in line 24 of your Dockerfile. You should not change permissions in a Dockerfile, docker keeps the permission that file has (you should run the chmod outside).

Anyway, why are you using docker for the unit tests? I think there are many things that I do not understand in the question so please, point me to your repository and branch where this action is so I can take a look.

**Reply by dariogmori** [reply]
Seems that it was a temporal error with SonarCloud that has been resolved, since we have not touched any files and now running the actions work. Sorry for the inconveniences.
